pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -150,6 +150,9 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 
 				rbacv1helpers.NewRule(read...).Groups(batchGroup).Resources("jobs/status", "cronjobs/status").RuleOrDie(),
 
+				rbacv1helpers.NewRule(read...).Groups(configGroup).Resources("apiservers", "authentications", "builds", "clusteroperators", "clusterversions",
```

Comment:
Add a unit test that requires this to be in sync with all resources registered in the config group.  We also want a matching rule for the operator group.

Run `UPDATE_BOOTSTRAP_POLICY_FIXTURE_DATA=true hack/test-go.sh pkg/cmd/server/bootstrappolicy` to update bootstrap YAML.

Reply:
> Add a unit test that requires this to be in sync with all resources registered in the config group. We also want a matching rule for the operator group.
> 
> Run `UPDATE_BOOTSTRAP_POLICY_FIXTURE_DATA=true hack/test-go.sh pkg/cmd/server/bootstrappolicy` to update bootstrap YAML.

Not visible because they come from the install path.